### PR TITLE
feat: add `Window::is_resizable`

### PR DESCRIPTION
**Pull request by amrbashir**
- [X] Tested on all platforms changed
- [X] Added an entry to `CHANGELOG.md` if knowledge of this change could be valuable to users
- [X] Updated documentation to reflect any user-facing changes, including notes of platform-specific behavior
- [X] Created or updated an example program if it would help users understand this functionality
- [X] Updated [feature matrix](https://github.com/rust-windowing/winit/blob/master/FEATURES.md), if new features were added or implemented

see #2169 for context

### CODE REVIEW

**Review by madsmtm — CHANGES_REQUESTED** [review]
Nice, thanks!

Can we document this in `set_resizable` (I assume these are directly correlated, as in, the value set in `set_resizable` is reflected in `is_resizable`)?

**Comment by filnet** [comment]
Also move `is_resizable` just after the corresponding `set_` function so it matches other `set_` / `is_` couples.
Ideally do the same for the similar PRs you created (visible and decorated)

File: src/platform_impl/web/window.rs [hunk 1]
```diff
@@ -246,6 +246,10 @@ impl Window {
         false
     }
 
+    pub fn is_resizable(&self) -> bool {
```
Comment (filnet) [inline src/platform_impl/web/window.rs:249]: some are `#[inline]` while others not. Is there a logic ?

Comment (amrbashir) [reply src/platform_impl/web/window.rs:249]: Just a typo on my end. I just follow the convention of the platforms I didn't implement

**Comment by amrbashir** [comment]
> Nice, thanks!
> 
> Can we document this in `set_resizable` (I assume these are directly correlated, as in, the value set in `set_resizable` is reflected in `is_resizable`)?

Seems like `set_maximized` and `is_maximized` don't link to each other and are just written one after another, should I follow this convention or should I link each setter with its getter (including `set_maximized` and `is_maximized`) ?

**Review by madsmtm — APPROVED** [review]
> written one after another

That's good enough for me, thanks!

**Review by kchibisov — APPROVED** [review]

**Pull request merged**